outline.txt

```diff
@@ -15,7 +15,7 @@
         -------------  
       /               \
 ---------                   ---------           ---------              ---------           ---------            
-ENCRYPTOR                   DECRYPTOR             KEY                   OFFSET              CRACK
```

Comment:
this only serves as a draft of the OOP structure of the project and will be removed upon completion